runtime/stdlib/contracts/test.cdc

```diff
@@ -95,6 +95,38 @@ pub contract Test {
         pub fun useConfiguration(_ configuration: Configuration) {
             self.backend.useConfiguration(configuration)
         }
+
+        /// Returns all the logs from the blockchain, up to the calling point.
+        ///
+        pub fun logs(): [String] {
+            return self.backend.logs()
+        }
+
+        /// Returns the service account of the blockchain. Can be used to sign
+        /// transactions with this account.
+        ///
+        pub fun serviceAccount(): Account {
+            return self.backend.serviceAccount()
+        }
+
+        /// Returns all events emitted from the blockchain.
+        ///
+        pub fun events(): [AnyStruct] {
```

Comment:
Given this API is guaranteed not to work because of https://github.com/onflow/cadence-tools/pull/113#discussion_r1205979891, maybe let's not expose it in the `Blockchain` yet (we can keep those in other places, like in the `BlockchainBackend`, etc.)?

Reply:
OR to save the trouble of having to do another release to get this in, maybe add a note saying this is still unimplemented, and at the implementation side (cadence-tools), we can return an empty array for now (better than panicing the tests).

Reply:
Actually, ^this can be part of the documentation, hence not a blocker for the PR.